tile.py

```diff
@@ -0,0 +1,20 @@
+
+class Tile():
+    def __init__(self, row, column, color):
+        self.row = row
+        self.column = column
+        self.piece = None
+        self.color = color
+        self.gui_tile = None
+    
+    def addPiece(self, piece):
+        self.piece = piece
+        
+    def removePiece(self):
+        toRemove = self.piece
+        self.piece = None
+        return toRemove
+    
+    def isOccupied(self):
+        return self.piece != None
+tile_colors = ['white', 'black']    
```

Comment:
According to the `__init__` the color is being passed as an argument from the App when each tile is created, and this list is exported to be used there. Also it's already present in the App class, so you can use it from there and pass it to an hypothetic method of game, like this: `self.Game.start(self.tile_colors)`. Other than that remember to leave an EOF newline in every file before saving.